graphql_compiler/schema_transformation/merge_schemas.py

```diff
@@ -20,19 +21,41 @@
 )
 
 
-def merge_schemas(schema_id_to_ast):
-    """Check that input schemas do not contain conflicting definitions, then merge.
+CrossSchemaEdgeDescriptor = namedtuple(
+    'CrossSchemaEdgeDescriptor', (
+        'edge_name',  # str, name used for the corresponding in and out fields
+        'outbound_side',  # FieldReference for the outbound (source) field
```

Comment:
nit: FieldReferenced namedtuple. (I know that it's a bit obvious that it is a namedtuple here, but just to be consistent with the comment I left below).